rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -196,6 +196,13 @@ def _precompiled_apple_resource_bundle_impl(ctx):
         outputs = [output_bundle_dir],
     )
 
+    xccurrentversions = [
+        f
+        for resource_files in ctx.attr.resources
+        for f in resource_files.files.to_list()
+        if f.path.count("xccurrentversion")
```

Comment:
Are you using this approach vs. endswith or in or similar for a specific reason?

Reply:
nope, actually `endswith` or `extension` is probably a better check, will update

Reply:
https://github.com/bazel-ios/rules_ios/pull/747/commits/2eb797e05ba9f2416b6b3a8f7923c7b5d041bebc